airbyte-server/src/main/java/io/airbyte/server/apis/ConfigurationApi.java

```diff
@@ -362,6 +363,11 @@ public void revokeSourceDefinitionFromWorkspace(final SourceDefinitionIdWithWork
     });
   }
 
+  @Override
```

Comment:
Nit: could you add a comment about where the implementation lives?

Reply:
oh yes! will do that real quick in a follow up